tests/tests/src/belt_list_test.res

```diff
@@ -171,27 +171,27 @@ describe(__MODULE__, () => {
   test("removeAssoc", () => {
     let eqx = (x, y) => (x: int) == y
 
-    ok(__LOC__, N.hasAssoc(list{(1, "1"), (2, "2"), (3, "3")}, 2, \"="))
```

Comment:
Why is this changed in the tests?

Reply:
An operator `\"op"` gives access to the internal representation of the operator.
So we always had access to the OCaml representation. Now, it has to be the ReScript representation.

Reply:
Ah right 🤦